app/src/main/java/com/stevesoltys/seedvault/ui/storage/StorageRootResolver.kt

```diff
@@ -141,6 +141,10 @@ internal object StorageRootResolver {
                 context.getDrawable(R.drawable.davx5)
             }
 
+            authority == AUTHORITY_ROUND_SYNC -> {
+                context.getDrawable(R.drawable.round_sync)
+            }
+
```

Comment:
Usually, we include an icon only for providers that we want users to be able to install manually during SUW, otherwise take the icon from the package manager directly. Is this the case here? Did I miss the part about opening a market:// link for the app?

Reply:
It would save us work to keep this as-is, since this package will one day be available in F-Droid